frontend/src/components/Board/RegularBoard/ParticipantsList/ParticipantCard.tsx/index.tsx

```diff
@@ -1,101 +1,154 @@
+    const {
+      addAndRemoveBoardParticipants: { mutate },
+    } = useParticipants();
+
+    const router = useRouter();
+    const boardId = router.query.boardId as string;
+
+    const isMemberResponsible = member.role === BoardUserRoles.RESPONSIBLE;
 
-  const router = useRouter();
-  const boardId = router.query.boardId as string;
+    const handleRemove = () => {
+      if (!member._id) return;
 
-  const handleRemove = () => {
-    if (!member._id) return;
+      const boardUsersToUpdate: UpdateBoardUser = {
+        addBoardUsers: [],
+        removeBoardUsers: [member._id],
+        boardId,
+      };
 
-    const boardUsersToUpdate: BoardUserAddAndRemove = {
-      addBoardUsers: [],
-      removeBoardUsers: [member._id],
-      boardId,
+      mutate(boardUsersToUpdate);
     };
 
-    mutate(boardUsersToUpdate);
-  };
-  return (
-    <Flex css={{ flex: '1 1 1' }} direction="column">
-      <Flex>
-        <InnerContainer
-          align="center"
-          elevation="1"
-          justify="between"
-          css={{
-            position: 'relative',
-            flex: '1 1 0',
-            py: '$22',
-            maxHeight: '$76',
-            ml: 0,
-          }}
-        >
-          <Flex align="center" css={{ width: '100%' }} gap="8">
-            <Icon
-              name="blob-personal"
-              css={{
-                width: '32px',
-                height: '$32',
-                zIndex: 1,
-                opacity: isOpen ? 0.2 : 1,
-              }}
-            />
-            <Flex align="center" gap="8" justify="start" css={{ width: '50%' }}>
-              <StyledMemberTitle>
-                {`${member.user.firstName} ${member.user.lastName}`}
-              </StyledMemberTitle>
-            </Flex>
-            {!isBoardCreator && (
-              <Flex align="center" css={{ width: '50%' }} justify="end">
-                <Tooltip content="Remove participant">
-                  <Flex justify="end">
-                    <IconButton
-                      onClick={handleRemove}
-                      css={{
-                        '&:hover': {
-                          cursor: 'pointer',
-                        },
-                      }}
-                    >
-                      <Icon
-                        name="trash-alt"
-                        css={{
-                          color: '$primary400',
-                          size: '$20',
-                        }}
-                      />
-                    </IconButton>
+    const updateIsResponsibleStatus = (checked: boolean) => {
+      let role: BoardUserRoles;
+      if (checked) {
+        role = BoardUserRoles.RESPONSIBLE;
+      } else {
+        role = BoardUserRoles.MEMBER;
+      }
+
+      const boardUserToUpdate: UpdateBoardUser = {
+        addBoardUsers: [],
+        removeBoardUsers: [],
+        boardUserToUpdateRole: {
+          ...member,
+          role,
```

Comment:
```suggestion
          role: checked ? BoardUserRoles.RESPONSIBLE : BoardUserRoles.MEMBER,
```
you can also do the role verification here